docs/antrea-network-policy.md

```diff
@@ -1777,3 +1777,10 @@ Similar RBAC is applied to the ClusterGroup resource.
 - NetworkPolicies are connection/flow oriented and stateful. They apply to
   connections, instead of individual packets, which means established connections
   won't be blocked by new rules.
+- For hairpin service traffic, when a Pod initiates traffic towards the service it provides,
+  and the same Pod is selected as the Endpoint, NetworkPolicies will consistently permit
+  this traffic during ingress enforcement if Antrea-Proxy is enabled. However, when Antrea-Proxy
```

Comment:
s/Antrea-Proxy/AntreaProxy/ to follow the writing convention.

Reply:
Done.